### ISSUE TITLE
Last Persistend id

### ISSUE THREAD

**Issue by hotmeatballs** [MatthieuLemoine/push-receiver#26]
Hello!
Is it possible to get all notification from one last persistent id?
Thank you!

**Comment by hotmeatballs** [comment]
`// Id of the last notification received
      clientEvent          : [],
      receivedPersistentId : this._persistentIds,`
I mean is it possible to send only one(!) last viewed persistent id?

**Comment by MatthieuLemoine** [comment]
I don't think so telling google that you have received a message does not mean you have received the previous ones.
